mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -294,7 +294,98 @@ bool GlAnimationOverlayCalculator::LoadAnimationAndroid(
       return false;
     }
 
+    // Use the vertices in order to generate normals for shader usage.
     triangle_mesh.normals.reset(new float[lengths[0]]);
+    float vertex_normals[lengths[0]];
+    float triangle_surface_normals[lengths[0]];
+    int len = lengths[0];
+
+    // Compute every triangle surface normal and store them for later use.
```

Comment:
The vertices are not stored in any particular order, so this normal calculation code doesn't really work (some vertices will be used in only 1 triangle, some vertices may be used in 10 triangles, etc.).  The triangle_mesh.triangle_indices list contains the indices into the vertex list for forming the actual triangles of our model.  So any normals need to be calculated over those.